src/it/resources/org/checkstyle/suppressionxpathfilter/anoninnerlength/SuppressionXpathRegressionAnonInnerLength.java

```diff
@@ -0,0 +1,15 @@
+package org.checkstyle.suppressionxpathfilter.anoninnerlength;
+
+import java.util.Comparator;
+
+public class SuppressionXpathRegressionAnonInnerLength {
```

Comment:
All input files should start with `Input`.

Reply:
I am not remember why ... but we do not do this notations in this part of code.
Examples:
https://github.com/checkstyle/checkstyle/tree/master/src/it/resources/org/checkstyle/suppressionxpathfilter/classmemberimpliedmodifier
https://github.com/checkstyle/checkstyle/tree/master/src/it/resources/org/checkstyle/suppressionxpathfilter/emptyforiteratorpad

Do you remember why ? We definitely did this on purpose.

Reply:
non of existing is named like "InputXxxxxxxx" ....
@timurt , do you remember why we did not followed general rule for input test java files ?

@rnveach , I think it should not be blocker for this PRs, as find a reason, or no reason, we can improve in next PRs.

Reply:
PR is merged, but if @timurt share reason, it would be good.

Reply:
@romani it seems that idea was to separate tests and inputs for `Checks` and for `Xpath Generator` I guess

I do not actually remember whole context, but please read this [discussion](https://github.com/checkstyle/checkstyle/issues/5648#issuecomment-386111791), I think it all came from there :)